src/objective-c/GRPCClient/GRPCCall+Tests.h

```diff
@@ -63,4 +63,19 @@
  * cache.
  */
 + (void)resetHostSettings;
+
+/**
+ * Enables logging of op batches. Memory consumption increases as more ops are logged.
+ */
++ (void)enableOpBatchLog:(BOOL)enabled;
+
+/**
+ * Add an op batch to log.
+ */
++ (void)addOpBatchToLog:(NSArray *)batch;
+
+/**
+ * Obtain the logged op batches. Invoking this method will clean the log.
+ */
++ (NSArray *)obtainAndCleanOpBatchLog;
```

Comment:
We might also be able to guard them (and the call to `addOpBatchToLog`) with some macro that we define for our tests, so that users don't get any performance penalty.

And I think we can make `addOpBatchToLog` actually private, as the tests don't have to call it.